static/manifest.json

```diff
@@ -18,34 +18,21 @@
   "content_scripts": [
     {
       "all_frames": true,
-      "js": [
-        "contentscript.js"
-      ],
-      "matches": [
-        "http://*/*",
-        "https://*/*"
-      ],
+      "js": ["contentscript.js"],
+      "matches": ["http://*/*", "https://*/*"],
```

Comment:
can u revert all the formatting changes?

Reply:
Reverted